decoder/gym.go

```diff
@@ -102,7 +103,7 @@ type Gym struct {
 //FROM information_schema.columns
 //WHERE table_schema = 'db_name' AND table_name = 'tbl_name'
 
-func getGymRecord(ctx context.Context, db db.DbDetails, fortId string) (*Gym, error) {
+func GetGymRecord(ctx context.Context, db db.DbDetails, fortId string) (*Gym, error) {
```

Comment:
why do these now need to be public?

Reply:
When a gym/stop is cleared I only have the id, but not gym object in general. 

And I need Gym object to access the must have props for hook... Accessing cache might be too less information, that's why I need to access db row too if not in cache

Reply:
But from where? Looks like its the same module so has access anyway

Reply:
🤔 have to try, I think it was not available in code completion